OutOfSchool/OutOfSchool.WebApi.Tests/Services/Database/WorkshopServiceTests.cs

```diff
@@ -235,6 +238,77 @@ public void Update_WhenClassIdIsInvalid_ShouldThrowArgumentOutOfRangeException()
 
         #endregion
 
+        #region UpdateStatus
+        [Test]
+        public async Task UpdateStatus_FromOpenToClosed_WhenEntityIsValid_ShouldReturnUpdatedEntity()
+        {
+            // Arrange
+            var workshopStatusDtoMock = WithWorkshopsList().
+                FirstOrDefault(w => w.Status == WorkshopStatus.Open && w.ProviderOwnership == OwnershipType.Private);
```

Comment:
Period at the end of the line is a bit confusing. Please move it to the beginning of the next line. Do the same in the code below.